data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_WORDS.fct

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Function Name="ASSEMBLE_DWORD_FROM_WORDS" Comment="">
+	<Identification Standard="61499-1" Description="Copyright (c) 2024 HR Agrartechnik GmbH    &#10;    &#10;This program and the accompanying materials are made     &#10;available under the terms of the Eclipse Public License 2.0     &#10;which is available at https://www.eclipse.org/legal/epl-2.0/     &#10;     &#10;SPDX-License-Identifier: EPL-2.0" >
+	</Identification>
+	<VersionInfo Version="1.0" Author="franz" Date="2024-02-22">
+	</VersionInfo>
+	<CompilerInfo>
+	</CompilerInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="REQ" Type="Event" Comment="">
+				<With Var="IN_00"/>
+				<With Var="IN_01"/>
+			</Event>
+		</EventInputs>
+		<EventOutputs>
+			<Event Name="CNF" Type="Event" Comment="">
+				<With Var=""/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN_00" Type="WORD" Comment="WORD 00"/>
+			<VarDeclaration Name="IN_01" Type="WORD" Comment="WORD 01"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="" Type="DWORD" Comment=""/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION ASSEMBLE_DWORD_FROM_WORDS : DWORD
+
+VAR_INPUT
+	IN_00 : WORD;    // WORD 00
+	IN_01 : WORD;    // WORD 01
+END_VAR
+
+// VAR_OUTPUT
+// OUT : DWORD; // DWORD Output
+// END_VAR
```

Comment:
remove this please, it gives no Information. 
please remove all 3 lines. 
@MoritzO02